app/templates/application.hbs

```diff
@@ -1,18 +1,15 @@
-{{set-body-class "deprecations deprecations_v2 responsive"}}
-
-{{#es-header}}
-
- {{#es-navbar}}
-    <form class="search-form">
-      {{search-input}}
-    </form>
-  {{/es-navbar}}
-{{/es-header}}
-
-<div id="content-wrapper" class="content-wrapper">
-  {{outlet}}
-</div>
-
-{{es-footer}}
+<EsHeader>
+  <form class="search-form">
+    <SearchInput />
+  </form>
+</EsHeader>
+
+<main>
+  <div class="container">
+    {{outlet}}
+  </div>
+</main>
+
+<EsFooter />
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/custom-elements/1.0.0/custom-elements.min.js"></script>
```

Comment:
Do we know what this is here for?

Reply:
It appears that it's a script that helps avoid Cloudflare apps from namespace collisions with the website they are installed on. 
https://www.cloudflare.com/apps/developer/docs/techniques/styles
I'm unaware of a Cloudflare app being installed on the Deprecations app so we probably don't need it. Removing it doesn't appear to break anything so I'll remove it for now.

Thanks for finding this!